### ISSUE TITLE
Bold text at block beginning results in hanging left indentation

### ISSUE THREAD

**Issue by straight-shoota** [postmodern/kramdown-man#14]
Bold text at the beginning of a text block creates a hanging left indentation (`.HP`) in roff output:
```md
**shards** requires the presence of a `shard.yml` file in the project
folder (working directory). This file describes the project and lists
dependencies that are required to build it.
```
```roff
.HP
\fBshards\fP requires the presence of a \fBshard.yml\fR file in the project
folder (working directory)\. This file describes the project and lists
dependencies that are required to build it\.
```

That seems unexpected to me. And `.HP` is deprecated.

**Comment by postmodern** [comment]
The `.HP` is added before regular paragraphs, not indented paragraphs where everything after the first line should be indented.
https://github.com/postmodern/kramdown-man/blob/0d5ee35476ead5b2ab1be0e8241581345b4d579e/lib/kramdown/converter/man.rb#L706

Recommendations on a better Roff modifier are welcomed.

**Comment by postmodern** [comment]
This is due to the code which tries to identify paragraphs where the first line isn't indented, but the following lines are.
https://github.com/postmodern/kramdown-man/blob/0d5ee35476ead5b2ab1be0e8241581345b4d579e/lib/kramdown/converter/man.rb#L706

I'm actually thinking of removing that code, in favor of using actual markdown definition lists to represent option and argument definitions.
https://kramdown.gettalong.org/syntax.html#definition-lists

**Comment by postmodern** [comment]
I changed the `convert_p` method to use `.TP` for multi-line paragraphs that begin with bold/codespan/em, and `.PP` for regular paragraphs. This is for rendering options or argument descriptions:

```
`-o`, `--opt` *VALUE*
  Description of option here.
  
*ARG*
  Description of the argument here.
```

In `kramdown-man` 1.0.0 I will switch to using markdown definitions, and render paragraphs normally with `.PP` macros.

```
`-o`, `--opt` *VALUE*
: Description of option here.
  
*ARG*
: Description of the argument here.

Regular paragraph here.

*Paragraph* that starts with an italic word that won't be confused as an option or argument definition.
```

**Comment by postmodern** [comment]
This should be fixed now in the [1.0.0 branch](https://github.com/postmodern/kramdown-man/tree/1.0.0).

```markdown
**shards** requires the presence of a `shard.yml` file in the project
folder (working directory). This file describes the project and lists
dependencies that are required to build it.
```

```roff
.\" Generated by kramdown-man 1.0.0
.\" https://github.com/postmodern/kramdown-man#readme
.PP
\fBshards\fP requires the presence of a \fBshard.yml\fR file in the project
folder (working directory)\. This file describes the project and lists
dependencies that are required to build it\.
```
